src/main/java/com/blog/web/repository/UserRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.blog.web.repository;
+
+import com.blog.web.models.UserEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<UserEntity, Long> {
+    UserEntity findByEmail(String email);
+    UserEntity findByUsername(String username);
+
```

Comment:
why is there a new-line for here seems like just a set of functions no need for seperation